react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+let i = 1;
+export default class HotButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: i };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState(prevState => ({
+      isClicked: i++
+    }));
```

Comment:
you should not create a variable outside of the class for state. this limits you to only using the component once. if you put two custom buttons on the page, you'll see what i mean

```jsx
root.render(
  <div>
    <HotButton />
    <HotButton />
  </div>
)
```